src-docs/src/views/text/text.js

```diff
@@ -19,9 +19,10 @@ export default () => (
       <h1>This is Heading One</h1>
       <p>
         Veniam et quis in dolor aliqua dolor laboris nostrud nostrud Lorem.
-        <a href="#">unfashionable</a> Irure et et adipisicing eu mollit. Ullamco
-        laborum cillum ea id occaecat cupidatat ex dolor consequat ex mollit do.
-        Irure commodo incididunt reprehenderit deserunt cillum quis ad nostrud!{' '}
+        <a href="https://oui.opensearch.org/latest/">unfashionable</a> Irure et
+        et adipisicing eu mollit. Ullamco laborum cillum ea id occaecat
+        cupidatat ex dolor consequat ex mollit do. Irure commodo incididunt
+        reprehenderit deserunt cillum quis ad nostrud!{' '}
```

Comment:
<img width="600" alt="Screenshot 2023-04-18 at 09 30 09" src="https://user-images.githubusercontent.com/17855243/232663816-6136e071-9aa4-4f28-b2e4-47da5a10bb99.png">

Reply:
Did the . after "Lorem" get added by accident or was that already there? Apologies on increasing scope, but lets go ahead and link any other word in this paragraph and remove the "unfashionable" - I think this was leftover from some proactive quick changes to OUI Docs before release to get rid of old EUI/Elastic/Kibana content.

Reply:
You are right, this word was taken from the previous text. Made the changes according to the comment

<img width="600" alt="Screenshot 2023-04-19 at 10 10 39" src="https://user-images.githubusercontent.com/17855243/232965024-93a2879e-2ceb-42bd-a976-552623e2b2d3.png">